### PR TITLE
RHEL release

### PR DESCRIPTION
**Pull request by surajsbharadwaj**
### Description

Please include a summary of the change and which issue is fixed. Also, please include the motivation and context for the change. List any dependencies that are required for this change.

**Check the relevant boxes:**
- [ ] Bug fix (nonbreaking change that fixes an issue)
- [x] New feature (nonbreaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Examples or tests (addition or updates of examples or tests)
- [x] Documentation update
- [ ] CI related update (pipeline, etc.)

### Checklist

- [ ] If relevant, a test for the change has been added or updated as part of this PR.
- [ ] If relevant, documentation for the change has been added or updated as part of this PR.

### Merge

- Merge using "Squash and merge".
- Make sure to use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents. The commit message determines whether a new version of the modules needs to be released, and if so, which semver number to use).

### CODE REVIEW

**Comment by ocofaigh** [comment]
@surajsbharadwaj Can you sync up with @SirSpidey  - he was also reviewing variable descriptions for this module, and had initially opened https://github.com/terraform-ibm-modules/terraform-ibm-powervs-infrastructure/pull/73 so might be a duplicate effort here

**Comment by surajsbharadwaj** [comment]
@ocofaigh i know he had reviewed it. I took his inputs and updated few things , including support for rhel as well

File: examples/ibm-catalog/standard-solution/variables.tf [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+  description = "IBM Cloud Schematics workspace ID of an existing Secure infrastructure on VPC for regulated industries with VSIs deployment. If you do not yet have an existing deployment, click [here](https://cloud.ibm.com/catalog/content/slz-vpc-with-vsis-a87ed9a5-d130-47a3-980b-5ceb1d4f9280-global#create) to create one. Please note: a [specific configuration](https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-powervs-infrastructure/main/examples/ibm-catalog/standard-solution/slz_json_configs_for_powervs/vpc_landscape_config.json) is needed for the deployment. Copy and paste that configuration from the link into the `override_json_string` deployment value."
```
Comment (SirSpidey) [inline examples/ibm-catalog/standard-solution/variables.tf:2]:
```suggestion
  description = "IBM Cloud Schematics workspace ID of an existing Secure infrastructure on VPC for regulated industries with VSIs deployment. If you do not yet have an existing deployment, click [here](https://cloud.ibm.com/catalog/content/slz-vpc-with-vsis-a87ed9a5-d130-47a3-980b-5ceb1d4f9280-global#create) to create one. When you deploy this module, use this [configuration](https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-powervs-infrastructure/main/examples/ibm-catalog/standard-solution/slz_json_configs_for_powervs/vpc_landscape_config.json). Copy the configuration from the link into the `override_json_string` deployment value."
```

File: examples/ibm-catalog/standard-solution/variables.tf [hunk 2]
```diff
@@ -18,7 +18,7 @@ variable "powervs_resource_group_name" {
 }
 
 variable "ssh_private_key" {
-  description = "Private SSH key used to login to IBM PowerVS instances. Should match to uploaded public SSH key referenced by 'ssh_public_key'. Entered data must be in [heredoc strings format] (https://www.terraform.io/language/expressions/strings#heredoc-strings). The key is not uploaded or stored. Read [here] more about SSH keys in IBM Cloud (https://cloud.ibm.com/docs/vpc?topic=vpc-ssh-keys)."
+  description = "Private SSH key used to login to IBM PowerVS instances. Should match to uploaded public SSH key referenced by 'ssh_public_key'. Entered data must be in [heredoc strings format](https://www.terraform.io/language/expressions/strings#heredoc-strings). The key is not uploaded or stored. Read more about SSH keys in IBM Cloud [here](https://cloud.ibm.com/docs/vpc?topic=vpc-ssh-keys)."
```
Comment (SirSpidey) [inline examples/ibm-catalog/standard-solution/variables.tf:21]:
```suggestion
  description = "Private SSH key used to login to IBM PowerVS instances. Should match to uploaded public SSH key referenced by 'ssh_public_key'. Entered data must be in [heredoc strings format](https://www.terraform.io/language/expressions/strings#heredoc-strings). The key is not uploaded or stored. For more information about SSH keys, see [SSH keys](https://cloud.ibm.com/docs/vpc?topic=vpc-ssh-keys)."
```

**Review by SirSpidey — COMMENTED** [review]
Couple of suggestions. I'll be out today, so I don't need to re-review if this is getting merged today.

**Review by Beilmann — DISMISSED** [review]

**Review by daniel-butler-irl — APPROVED** [review]

**Pull request merged**